apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -581,6 +594,11 @@
             }
         }
 
+        function handleCredentialResponse(response) {
+            document.getElementById("googleOneTapForm").elements.credential.value = response.credential;
```

Comment:
Try to simplify,
`$('#credential').val(response.credential)`

Reply:
Fixed 73f0c7d1d762facf2161caf8e55de61c9379dc20